assets/scss/core/_typography.scss

```diff
@@ -107,18 +107,20 @@ small {
   line-height: $dp-font-small-lineheight;
 }
 
-.link-green {
+a {
```

Comment:
I love it, but, how could you ensure that this will not affect unwanted anchors?

Reply:
It's already implemented, only I moving the styles in the tipography file, but all anchors is green.

Reply:
I am not sure, please test it in the webapp before merging.

Reply:
I agree with @andresmoschini. This change is going to affect all links independently of it's location. This should be reverted since it surely will affect other links in other places different from the webapp in where the library could be used, @IgnacioRodrigues

Reply:
@IgnacioRodrigues I agree with @andresmoschini. This change should be reverted since all links would be affected despite it's location.